Docker/start-container.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+#Set user and group
+umask 0002
+PUID=${PUID:-id -u squeezeboxserver}
+PGID=${PGID:-id -g users}
+
+usermod -o -u "$PUID" squeezeboxserver
+groupmod -o -g "$PGID" users
+
+umask 0002
+
+#Add permissions
+chown -R squeezeboxserver:users /config /playlist
+
+/usr/bin/perl /lms/slimserver.pl --user squeezeboxserver --prefsdir /config/prefs --logdir /config/logs --cachedir /config/cache
```

Comment:
What was the reason you removed supervisor? There is nothing controlling the service now.

Reply:
Is there a need for a supervisor? I never had any kind of supervisors in my LMS installations.

Reply:
For Docker installations it is. Most LMS dockers use images that already have a supervisor that is why you don't see them in the build. 

There should be a process that keeps an eye on the running services as if the break it will try to restart them.

Reply:
I'm using an image for a while without a supervisor without problems. Looking up supervisord it is most of the time used for multiple services in one container, but that is here not really the case. Also, without supervisord the logs of LMS get directly feeded to the docker log. But if there is really a good reason for supervisor we should add it.

Reply:
I have also used an image without a supervisor for a couple of years now without any problems.

In general, I think supervisors in single-process containers are a mistake.  They use additional memory, can confuse signal handling/propagation, and often hide the fact that a service is breaking - it's better to have the container exit, know that you have a problem, and fix the problem.  If you really need the process to be restarted no matter what, you can easily wrap your `docker run` in a systemd unit file (or some other supervisor running *outside* of docker).

Reply:
I'm look at the bigger picture to prepare a good base for future. For example for users that will need airplay we will have to run extra service.

Community would then need minor mod to "official" one as majority will be prepared.

Reply:
For me LMS has been so stable I never saw the need to have a supervisor. And on the one system where we actually have something (Debian), users are asking to get rid of it, for the reasons @tmancill mentioned: if things crash you should know about it. A supervisor which just tries to restart a failing service would often hide the problems. I'd rather know about them.